source/sync/configure/enable-sync.txt

```diff
@@ -714,46 +864,63 @@ Enable Flexible Sync
             :ref:`link-a-data-source` guide to link it before
             you continue.
 
-            You'll need the cluster's service configuration file to configure Device Sync. You
-            can find the configuration file by listing all services through the Admin API:
-
             .. note:: Authenticating Your Request with an Access Token
 
               To authenticate your request to the App Services Admin API, you need a valid and
               current authorization token from the MongoDB Cloud API. Read the
               :admin-api-endpoint:`API Authentication <section/Get-Authentication-Tokens>`
               documentation to learn how to acquire a valid access token.
 
+            You'll need the cluster's service configuration file to configure sync. You
+            can find the configuration file by :admin-api-endpoint:`listing all services 
+            through the Admin API <operation/adminListServices>`:
+
             .. code-block:: shell
 
                curl https://realm.mongodb.com/api/admin/v3.0/groups/{GROUP_ID}/apps/{APP_ID}/services \
                  -X GET \
                  -h 'Authorization: Bearer <Valid Access Token>'
 
-            Once you have the configuration, add the ``config.flexible_sync`` field with the
+            Identify the service whose configuration you need to update to enable
+            Sync. If you have accepted the default names when configuring 
+            your App, this should be a service whose ``name`` is ``mongodb-atlas``
```

Comment:
```suggestion
            your App, this should be a service whose ``name``
```

Reply:
I want to be very explicit about these values and don't mind being repetitive in this instance, so I'm going to leave this one in.